File: 1019.cpp
// André Nascimento
// GitHub: github.com/AndreNasci
// Linkedin: linkedin.com/in/andre-nascimento-a01095185

#include <bits/stdc++.h>

using namespace std;

int main()
{
    unsigned long N, hours, minutes;

    cin >> N;

    hours = N / 3600;
    minutes = N % 3600 / 60;

    cout << hours << ":" << minutes << ":" << N - hours*3600 - minutes*60 << endl;
    return 0;
}

File: Python/1019.py
"""
André Nascimento
GitHub: github.com/AndreNasci
Linkedin: linkedin.com/in/andre-nasci/
"""

seconds = int(input())

minutes = seconds // 60
seconds = seconds % 60

hours = minutes // 60
minutes = minutes % 60

print(f"{hours}:{minutes}:{seconds}")